src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -570,11 +563,34 @@ private IAccount GetAccountByUserFlow(IEnumerable<IAccount> accounts, string use
             foreach (var account in accounts)
             {
                 string accountIdentifier = account.HomeAccountId.ObjectId.Split('.')[0];
-                if (accountIdentifier.EndsWith(userFlow.ToLower()))
+                if (accountIdentifier.EndsWith(userFlow.ToLower(CultureInfo.CurrentCulture), StringComparison.InvariantCulture))
                     return account;
             }
 
             return null;
         }
+
+        internal /*for test only*/ string CreateRedirectUri()
```

Comment:
Do we want to keep /* for test only */ ?

Reply:
we do in msal .net. i guess for us to know why it's internal and not private. @bgavrilMS your thoughts on this? i htink you started this in msal.

Reply:
Ideally we should keep exposing internal stuff at a minimum. Is it possible to check the redirect URI on the `cca` object you create?

Reply:
The method which creates the cca does a lot and I'd rather just test this specific method. :/

---
In reply to: [420319718](https://github.com/AzureAD/microsoft-identity-web/pull/144#discussion_r420319718) [](ancestors = 420319718)